### PR TITLE
Add random color functionality

### PR DESCRIPTION
**Pull request by korbosoft**
gonna test this now

### CODE REVIEW

**Comment by korbosoft** [comment]
should be ready now

File: crimsobot/cogs/utilities.py [hunk 1]
```diff
@@ -169,9 +171,11 @@ async def ping(self, ctx: commands.Context) -> None:
         await msg.edit(content='<:ping:569954524932997122>...{:d}ms'.format(int(ping)))
 
     @commands.command()
-    async def color(self, ctx: commands.Context, hex_value: discord.Colour) -> None:
-        """Get color sample from hex value."""
-
+    async def color(self, ctx: commands.Context, hex_value: Optional[discord.Colour] = None) -> None:
+        """Get color sample from hex value, or generate random color if not given input."""
+        if hex_value is None:
```
Comment (h-anjru) [inline crimsobot/cogs/utilities.py:176]: Need whitespace here.

File: crimsobot/cogs/utilities.py [hunk 2]
```diff
@@ -169,9 +171,11 @@ async def ping(self, ctx: commands.Context) -> None:
...
+            hex_value = '#%06x' % randint(0, 0xFFFFFF)
+        print(str(hex_value))
```
Comment (h-anjru) [inline crimsobot/cogs/utilities.py:178]: Need to nix this print(); don't want it clogging up the terminal.

**Review by h-anjru — CHANGES_REQUESTED** [review]

**Review by h-anjru — APPROVED** [review]

**Review by h-anjru — APPROVED** [review]

**Review by h-anjru — CHANGES_REQUESTED** [review]
Whitespace line 175. Run flake8 before next commit.

**Review by h-anjru — APPROVED** [review]

**Pull request merged**